.github/workflows/platform.dependencies.yml

```diff
@@ -38,8 +38,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        parameterFilePaths:
-          ['artifacts.parameters.json', 'validation.parameters.json']
+        parameterFilePaths: ['validation.parameters.json']
```

Comment:
Will the artifacts RG and container registry be created directly by the publish job?

Reply:
Yes, they always were - and from the feedback I received it rather raised questions than resolve them :/ I hence decided to remove it from the dependency pipeline. Happy for feedback if we want to keep it regardless

Reply:
Fine for me